frontend/src/app/course/addCourseForm.jsx

```diff
@@ -22,7 +25,7 @@ const AddCourseForm = (): React.Node => {
         [dispatch],
     );
 
-    return (
+    return isLoggedIn && authorities.length === 2 ? (
```

Comment:
@teodorvoicencu fixed some stuff and now you can use `.includes(AuthorityTypes.TEACHER)` instead of `.length === 2` (which is kinda bad practice). You should rebase with `master` to fetch the new changes.

Reply:
ok